Mathlib/Data/PSigma/Order.lean

```diff
@@ -153,15 +153,15 @@ instance denselyOrdered_of_noMaxOrder [Preorder ι] [∀ i, Preorder (α i)]
       exact ⟨⟨i, c⟩, right _ ha, right _ hb⟩⟩
 #align psigma.lex.densely_ordered_of_no_max_order PSigma.Lex.denselyOrdered_of_noMaxOrder
 
-instance densely_ordered_of_noMinOrder [Preorder ι] [∀ i, Preorder (α i)]
+instance denselyOrdered_of_noMinOrder [Preorder ι] [∀ i, Preorder (α i)]
```

Comment:
I believe the naming convention is that manual instance names should start with "inst", i.e.
```suggestion
instance instDenselyOrdered_of_noMinOrder [Preorder ι] [∀ i, Preorder (α i)]
```
(I'm not 100% sure on the casing; you may want to compare with other names/wait for another pair of eyes.)

Reply:
If I do this, there will be many more examples of such instances in this file alone

Reply:
Sure; mathlib is not always consistent (and I think a fix for that would be welcome - to be sure, I'd ask on zulip first).

I'm not asking you to change any other instances! But when you're renaming this one, I'd rather rename it to a "right" than a "less wrong" name.

Reply:
Personally, I would find the resulting inconsistency with e.g. denselyOrdered_of_noMaxOrder even more confusing